hermes/serializers.py

```diff
@@ -138,59 +138,246 @@ class Meta(BaseTargetSerializer.Meta):
         fields = BaseTargetSerializer.Meta.fields + ['messages',]
 
 
-def validate_date(date, date_format=None):
-    if date_format:
-        if 'jd' in date_format.lower():
-            try:
-                float(date)
-            except ValueError:
-                raise serializers.ValidationError({'date': _(f"Date: {date} does not parse. JD formatted dates must be a float value.")})
-        else:
-            try:
-                datetime.strptime(date, date_format)
-            except ValueError:
-                raise serializers.ValidationError({'date': _(f"Date: {date} does not parse based on provided date format: {date_format}.")})
-    else:
+def validate_date(date):
+    try:
+        float(date)
+    except ValueError:
         try:
             parse(date)
         except ValueError:
-            raise serializers.ValidationError({'date': _(f"Date: {date} does not parse with dateutil.parser.parse. Please specify a date_format or change your date.")})
+            raise serializers.ValidationError({'date': _(f"Date: {date} does not parse with dateutil.parser.parse. Please specify the date in a standard format or as a JD.")})
```

Comment:
We don't distinguish between JD and MJD here, correct?
Any Float is accepted and stored as a string?

Reply:
yes, is that okay?

Reply:
I think it's fine, just something to remember if we ever try to convert these to date-time objects.